### ISSUE TITLE
session_id & more not being set

### ISSUE THREAD

**Issue by smlparry** [toothrot/amplitude-api#54]
Hey there

It seems that there are quite a few attributes that do not get set on the `AmplitudeAPI::Event` model when creating via `AmplitudeAPI::Event.new`

Given this hash:
```
{
 :event_id=>33785,
 :event_type=>"program_stage_viewed",
 :event_properties=>{"program_id"=>30, "program_title"=>"Program w/ Stages", "program_stage_id"=>1147, "program_stage_title"=>"Week 4"},
 :time=>123123123,
 :user_id=>10,
 :device_id=>"2b7b68cf-aed7-4387-ad07-6e3338c5fe7e",
 :session_id=>457,
 :platform=>"smartphone",
 :os_name=>"iOS",
 :os_version=>"13.2.3",
 :device_brand=>"Apple",
 :device_model=>"iPhone",
 :ip=>"172.19.0.1"
}
```

The resulting Event object only has the following attributes

```
#<AmplitudeAPI::Event:0x000055c69aecbcb8 
  @event_type="program_stage_viewed", 
  @event_properties={"program_id"=>30, "program_title"=>"Program w/ Stages", "program_stage_id"=>1147, "program_stage_title"=>"Week 4"}, 
  @time=123123123, 
  @user_id=10, 
  @device_id="2b7b68cf-aed7-4387-ad07-6e3338c5fe7e", 
  @platform="smartphone", 
  @ip="172.19.0.1">
```

Im just going off the documentation here: https://help.amplitude.com/hc/en-us/articles/360032842391#http-api-v2-events

Any chance this is a bug? Otherwise ill just have to use HTTParty or something

**Comment by AlvaroMaceda** [comment]
Hi, smlparry. I don't know if you still need it but I'm on it, it should be solved soon.

**Comment by AlvaroMaceda** [comment]
It should be solved in 0.3.0. Please, open a new issue if you continue having problems with that. Thank you for your report.
